src/layout/styles.js

```diff
@@ -29,7 +29,7 @@ const Body = styled.div`
 const BodyGeneral = styled.div`
   position: relative;
   flex: 1;
-  padding: 5rem 1.5rem 3rem;
+  padding: ${({ isMap }) => (isMap ? '5rem 0 0' : '5rem 1.5rem 3rem')};
```

Comment:
So i don't know What to do here, I removed this before but had to add it back in as Padding on the edges  just makes the map look ugly